pkg/xray/resource/resource_xray_binary_manager_release_bundle_v2_test.go

```
@@ -431,6 +431,110 @@ func TestAccBinaryManagerReleaseBundlesV2_project_full(t *testing.T) {
+			if err := createReleaseBundleV2(t, releaseBundleName, keyPairName, repoName, projectKey1, path, sha256); err != nil {
+				t.Fatalf("failed to create release bundle: %s", err)
+			}
+
+			if err := createReleaseBundleV2(t, releaseBundleName, keyPairName, repoName, projectKey2, path, sha256); err != nil {
+				t.Fatalf("failed to create release bundle: %s", err)
+			}
+		},
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy: func(*terraform.State) error {
+			if err := deleteReleaseBundleV2Version(t, releaseBundleName, projectKey1); err != nil {
+				return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 499) but it returns nil

(nilerr)